dali/operators/generic/slice/slice_attr.h

```diff
@@ -58,15 +72,15 @@ class SliceAttr {
     }
   }
 
-  void ProcessArguments(DeviceWorkspace &ws) {
+
+  void ProcessArguments(const MixedWorkspace &ws) {
```

Comment:
`MixedWorkspace` ??

Reply:
Yes, remember this is used by `ImageDecoderSlice` as well